tests/test_shell.py

```diff
@@ -269,8 +270,17 @@ def test_find(self):
         assert '{"a": 0}' in out
         assert '{"a": 2}' in out
 
+        job = project.open_job({"a": [0, 1, 0]})
+        msg = "python -m signac find a".split()
+        msg.append("[0, 1, 0]")
+        msg.append("--sp")
```

Comment:
Optional suggestion. Shorter but may be slightly less readable.
```suggestion
        msg = [*"python -m signac find a".split(), "[0, 1, 0]", "--sp"]
```